### ISSUE TITLE
Optifabric not working...

### ISSUE THREAD

**Issue by Goofball2** [Chocohead/OptiFabric#648]
Trying to run in 1.18.1. Fabric is working on its own. Optifabric is working but comes up with the error saying optifine jar isn't there. I download optifine 1.18.1 and put it in the mods folder and the game won't load. Please help!

**Comment by Goofball2** [comment]
The game crashed whilst rendering overlay
Error: java.lang.RuntimeException: Mixin transformation of net.optifine.CustomColors failed
Exit Code: -1073740791

**Comment by joshuaepstein** [comment]
Have you got FabricAPI installed as well?

**Comment by joshuaepstein** [comment]
Do you have any other mods

**Comment by Goofball2** [comment]
> Have you got FabricAPI installed as well?

No, I didn't think I needed it? I will go download it and see if it makes a difference

**Comment by Goofball2** [comment]
> Have you got FabricAPI installed as well?

I've done that and now it's in a loop of loading. With the red background and mojang studios. No other mods at the moment, want replay mod but need this working first

**Comment by Angelkoon** [comment]
i'm having the same issue

**Comment by Goofball2** [comment]
> i'm having the same issue

Hi, I found out that optifine H5 doesn't work (is not compatible), use 1.18.1 H4

**Comment by goose-vii** [comment]
Using H4 worked for me too-- thank you!

**Comment by harvanchik** [comment]
> Using H4 worked for me too-- thank you!

Close this issue now.

**Comment by ghost** [comment]
You can also check https://github.com/Chocohead/OptiFabric/pull/630 for the latest current testing version of Optifabric, although expect mod compatibility to change as support is added for more.

**Comment by Skywalker8510** [comment]
try using the latest version of OptiFabric
